ecowitt2mqtt/runtime.py

```diff
@@ -67,6 +67,15 @@ def __init__(self, ecowitt: Ecowitt) -> None:
             )
         )
 
+        @app.get("/health/liveness")
+        async def _async_liveness():
+            return { "status": "Online"}
```

Comment:
Is this always the value?

Reply:
Yes, these checks are for the kubes to know a service is alive or ready.. 
Live is hey i'm here and not crashing
Ready is I'm ready to server customers if they make a request. 

In your case your app is so small its really one and the same.